ideascube/mediacenter/tests/factories.py

```diff
@@ -4,12 +4,15 @@
 
 from ..models import Document
 
+class EmptyFileField(factory.django.FileField):
+    DEFAULT_FILENAME = None
 
 class DocumentFactory(factory.django.DjangoModelFactory):
     title = factory.Sequence(lambda n: "Test document {0}".format(n))
     summary = "This is a test summary"
     lang = settings.LANGUAGE_CODE
     original = factory.django.FileField()
+    preview = EmptyFileField()
```

Comment:
>> Seems the only alternative would have been to change all the existing tests so that they'd pass preview=None when instanciating the factory?
> 
> Yes :/ I've preferred to not change the API of the factory.

Well, the API wouldn't have changed, it's just that tests which really don't want a `preview` had to set it themselves.

In fact, with your change, you changed the behaviour of the factory for all the previously existing tests.

I do think this is a much more sensible default value though, since it's the same as the one from the model (as `preview` has `blank=True`).